tox.ini

```diff
@@ -0,0 +1,88 @@
+[tox]
+envlist = py27, lint, coverage-report
+skip_missing_interpreters = true
+
+[testenv]
+deps = -rrequirements.txt
+commands = pytest \
+  --cov=girder_worker \
+  --cov-config=.coveragerc.pytest \
+  # Surpress printing coverage, defer this to testenv:coverage-report
+  --cov-report= \
+  {posargs}
+
+# This report is used to merge coverage reports from testenv and from
+# the legacy tests run by ctest. When just running 'tox' it will print
+# the pytest coverage report. When running ctest && tox it will print
+# the combine coverage.
+[testenv:coverage-report]
+skip_install = true
+basepython = python2.7
+deps = coverage
+commands =
+  coverage combine
+  coverage report
+
+[testenv:lint]
+skip_install = true
+basepython = python2.7
```

Comment:
In the spirit of making GW Py3 compliant, should this be left out? I noticed running against Python3 produced 5 linting errors that could be resolved.

Reply:
Yes,  setting this basepython is a temporary measure.  it will be removed in a subsequent PR that makes the modern girder_worker code python2/3 compliant